### PR TITLE
Mike Harley

### PR DESCRIPTION
**Pull request by mharley12345**
Still have styling to do.

### CODE REVIEW

**Comment by mharley12345** [comment]
Updated Buttons

File: Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+    /* justify-content: flex-end; */
+    /* align-items: flex-end; */
+    width: 100%;
+    /* height: 472px; */
+  }
+  .number {
+    border-top: 2px solid gray;
+    border-right: 2px solid gray;
+    height: 95px;
+    width: 33%;
```
Comment (adammathieson) [inline Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css:16]:
```suggestion
    width: 33.33%;
```

Comment (adammathieson) [reply Project-II-Calculator/calculator/src/components/ButtonComponents/Button.css:16]: That'll do it

**Review by adammathieson — COMMENTED** [review]
Now that's a fine looking calculator! Nice work. I included just the one note to fix that pesky space.

**Pull request merged**